tools/topology/sof-tgl-max98373-rt5682.m4

```diff
@@ -62,6 +62,10 @@ define(`SMART_REF_CH_NUM', 4)
 define(`SMART_PCM_ID', 0)
 define(`SMART_PCM_NAME', `smart373-spk')
 
+# UUID related
+DECLARE_SOF_RT_UUID("Maxim DSM", maxim_dsm_comp_uuid, 0x0cd84e80, 0xebd3,
+                    0x11ea, 0xad, 0xc1, 0x02, 0x42, 0xac, 0x12, 0x00, 0x02);
+define(`SMART_UUID', maxim_dsm_comp_uuid)
```

Comment:
This means we will test Maxim DSM only with this tplg, are we ready for that?

Reply:
Hi Keyon,
Yes the basic DSM functionality is working, also the PR from maxim is merged & 2 are under review.

https://github.com/thesofproject/sof/pull/3248

Static library support + license/Readme information were included on the next PR.
https://github.com/thesofproject/sof/pull/3373

UUID change was applied to the next PR.
https://github.com/thesofproject/sof/pull/3372

Reply:
OK, then the PR here looks good to me, let's wait until the UUID change applied and then apply this here.